tests/integration/links_test.go

```diff
@@ -149,15 +149,13 @@ func testLinksAsUser(userName string, t *testing.T) {
 		"/settings/collaboration",
 		"/settings/branches",
 		"/settings/hooks",
-		// FIXME: below links should return 200 but 404 ??
-		//"/settings/hooks/git",
-		//"/settings/hooks/git/pre-receive",
-		//"/settings/hooks/git/update",
-		//"/settings/hooks/git/post-receive",
+		"/settings/hooks/git",
+		"/settings/hooks/git/pre-receive",
+		"/settings/hooks/git/update",
+		"/settings/hooks/git/post-receive",
 		"/settings/keys",
 		"/releases",
 		"/releases/new",
-		//"/wiki/_pages",
```

Comment:
I delete this one because only one of six repos has a wiki and return 200, otherwise just return 303(See others). It's not suitable in there.